plugins-builtin/src/main/resources/info_file_avro_schema.avsc

```diff
@@ -0,0 +1,144 @@
+{
+  "name": "infoFile",
+  "type": "record",
+  "namespace": "co.za.absa.dataquality.avro.schema-infofile",
+  "fields": [
+    {
+      "name": "datasetName",
+      "type": "string"
+    },
+    {
+      "name": "datasetVersion",
+      "type": "int"
+    },
+    {
+      "name": "reportDate",
+      "type": "string"
+    },
+    {
+      "name": "reportVersion",
+      "type": "int"
+    },
+    {
+      "name": "runStatus",
+      "type": "string"
+    },
+    {
+      "name": "metadata",
+      "type": {
+        "name": "metadata",
+        "type": "record",
+        "fields": [
+          {
+            "name": "sourceApplication",
+            "type": "string"
+          },
+          {
+            "name": "country",
+            "type": "string"
+          },
+          {
+            "name": "historyType",
+            "type": "string"
+          },
+          {
+            "name": "dataFilename",
+            "type": "string"
+          },
+          {
+            "name": "sourceType",
+            "type": "string"
+          },
+          {
+            "name": "version",
+            "type": "int"
+          },
+          {
+            "name": "informationDate",
+            "type": "string"
+          },
+          {
+            "name": "additionalInfo",
+            "type": {
+              "type": "array",
+              "items": {
+                "name": "detail",
+                "type": "record",
+                "fields": [
+                  {
+                    "name": "key",
+                    "type": "string"
+                  },
+                  {
+                    "name": "value",
+                    "type": "string"
+                  }
+                ]
```

Comment:
Why not use `map` here instead of `array`?

Reply:
This is something to discuss with the downstream team.